tests/blob_tx_test.go

```diff
@@ -49,11 +49,11 @@ func testBlobTx_WithBlobsIsRejected(t *testing.T, ctxt *testContext) {
 
 	// attempt to run tx
 	_, err = ctxt.net.Run(tx)
-	require.ErrorContains(err, "transaction type not supported")
+	require.ErrorContains(err, "oversized data")
```

Comment:
This change looks like it is breaking the previous intention of checking that the transaction type is rejected. The text indicates that now a different reason is causing the rejection, shadowing the originally check tested for.

It would be good to figure out why no suddenly the data is considered to big.